bonsai/Modules/TabBar/TabBar.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  TabBar.swift
+//  bonsai
+//
+//  Created by hoang on 11.12.2021.
+//
+
+import SwiftUI
+
+struct TabBar: View {
+
+   @State private var selection = 1
+
+   private let tabBarImages: [String] = [Asset.settingDisabled.name,
+                                         Asset.homeDisabled.name,
+                                         Asset.chartsDisabled.name]
+
+   private let chartsDisabled = UIImage(named: Asset.chartsDisabled.name) ?? UIImage()
+   private let chartsSelected = UIImage(named: Asset.chartsSelected.name) ?? UIImage()
+
+   private let homeDisabled = UIImage(named: Asset.homeDisabled.name) ?? UIImage()
+   private let homeSelected = UIImage(named: Asset.homeSelected.name) ?? UIImage()
+
+   private let settingsDisabled = UIImage(named: Asset.settingDisabled.name) ?? UIImage()
+   private let settingSelected = UIImage(named: Asset.homeSelected.name) ?? UIImage()
+
+   var body: some View {
+      VStack {
+         HStack {
+            ForEach(0..<3) { num in
+               Label {
+               } icon: {
+                  if num == 0 {
+                     Spacer()
+                     Spacer()
+                     Image(uiImage: selection == num ? settingSelected : settingsDisabled)
```

Comment:
It would be better to use template image and foreground color instead I think.

Because this way we will be able to customise selected color from code without any need to create new assets

Reply:
i tried to change foreground, but unsuccess, i think its related to tintcolor of image, need to research this moment:)